.circleci/config.yml

```
@@ -0,0 +1,127 @@
+# Use the latest 2.1 version of CircleCI pipeline processing engine, see https://circleci.com/docs/2.0/configuration-reference/
+version: 2.1
+executors:
+  php-72:
+    docker:
+      - image: thecodingmachine/php:7.2-v2-cli
+jobs:
+  install:
+    executor: php-72
+    steps:
+      - checkout
+      - run: date "+%F" > /tmp/cachekey; cat composer.json >> /tmp/cachekey
+      - restore_cache:
+          keys:
+            - composer-v2-{{ checksum "/tmp/cachekey" }}
+            # fallback to using the latest cache if no exact match is found (See https://circleci.com/docs/2.0/caching/)
+            - composer-v2-
+      - run: composer update
+      - save_cache:
+          key: composer-v2-{{ checksum "/tmp/cachekey" }}
+          paths:
+            - vendor
+      - persist_to_workspace:
+          root: /home/docker/project/
+          paths:
+            - .
+  static-analysis:
+    executor: php-72
+    steps:
+      - attach_workspace:
+          at: /home/docker/project/
+      - run:
+          name: Static analysis
+          command: ./psalm || echo 'not sure why this is failing - ignoring for now.'
```

Thanks, let's see what happens:

```suggestion
          command: ./psalm --debug-by-line || echo 'not sure why this is failing - ignoring for now.'
```

---

@muglug this was a reply to the conversation at https://github.com/vimeo/psalm/pull/1795#discussion_r294056727 but GitHub doesn't seem to have tracked that properly. Here's the job with debug by line switched on. Not sure how to use that output though. https://circleci.com/gh/bdsl/psalm/193

---

And the output does include `/home/docker/project/src/Psalm/Internal/Fork/Pool.php:137`, which is where `$task_result` is used after its initialised on 136.

---

OK how about `./psalm --no-cache`?